examples/todoapp/main.js

```diff
@@ -12,6 +13,10 @@ void async function main() {
     console.log("------ doc: ------\n", uc.doc())
 
     uc.authorize(user)
+    stopwatch.start("addOrUpdateItem")
```

Comment:
why are you using `stopwatch` here? shouldn't `elapsedTime` come from audit?

Reply:
Yes, I'll remove